kt/godot-library/src/test/kotlin/godot/core/GodotTest.kt

```diff
@@ -0,0 +1,22 @@
+@file:Suppress("NOTHING_TO_INLINE")
+
+package godot.core
+
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+
+inline fun checkTrue(value: Boolean) {
+    assertTrue(value)
+}
+
+inline fun checkFalse(value: Boolean) {
+    assertFalse(value)
+}
+
+inline fun checkMessage(value: Boolean, lazyMessage: () -> String) {
+    assertTrue(lazyMessage(), value)
+}
+
+inline fun checkFalseMessage(value: Boolean, lazyMessage: () -> String) {
+    assertFalse(lazyMessage(), value)
+}
```

Comment:
Not sure if I'm fond of these abstractions. For sure IMO assert is the better keyword. What is the reason for these abstractions?

Reply:
Because I wanted it to be easy to change whenever we switch to Junit 5

Reply:
ok for me